### PR TITLE
[Compute On-prem RN - LagrangeUpdate1]: Initial draft for RN blurbs

### PR DESCRIPTION
**Pull request by divyabhushan-okta**

### CODE REVIEW

File: compute/rn/release-information/release-notes-22-12-update1.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+[cols="1,4"]
+|===
+|Build
+|22.12.xxx
+
+|Codename
+|Lagrange, 22.12 Update 1
+|Release date
+|Jan 9, 2023
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-12-update1.adoc]: @Pubs-MV @divyabhushan I think this is not the release date

File: compute/rn/release-information/release-notes-22-12-update1.adoc [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+//Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+
+// LINK
+
+toc::[]
+
+=== New Features in Core
+// CWP-42365
+==== Filter Defender /api/vVERSION/defenders by TAS Foundation ID
+Introduces a new value,*tasFoundations*, in the *fields* query parameter to filter Defenders by TAS Foundation ID.
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-12-update1.adoc]: @divyabhushan I think it is worth mentioning that the filter is under Manage > Defenders > Deployed Defenders, as well as in the API /api/vVERSION/defenders

File: compute/rn/release-information/release-notes-22-12-update1.adoc [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  
+//CWP-41281
+==== ARM64 bit twistcli for MacOS platforms
+Download ARM64 Mac compatible version of twistcli under *Manage > System > Utilities*, or via */util/osx/arm64/twistcli* API.
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-12-update1.adoc:38]: @divyabhushan I would add a first line of "twistcli is now supported on ARM64 Mac machines." and then continue to "Download ARM64..."

File: compute/rn/release-information/release-notes-22-12-update1.adoc [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+//CWP-42198 - Rodrigo's validation needed
+==== Support for Talos Linux - container vulnerabilities and compliance
+
+Talos Linux is now supported as a host OS for orchestrator container Defenders.
+Defenders installed on Talos Linux will allow scanning vulnerability and compliance for the running containers, as well as performing registry scans.
+To deploy on Talos Linux cluster, use the new "Talos Linux deployment" toggle in the Defenders deployment page, or the new `--talos` flag in twistcli.
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-12-update1.adoc:45]: @divyabhushan @rcaballeromx 
We have some limitations for the TalOS support, can we please make sure it is described in the system requirements where we declare support for Talos? The limitations are described in the epic under "Docs guidelines" https://redlock.atlassian.net/browse/CWP-42198

File: compute/rn/release-information/release-notes-22-12-update1.adoc [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+//Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+
+// LINK
+
+toc::[]
+
+=== New Features in Core
+// CWP-42365
+==== Filter Defender /api/vVERSION/defenders by TAS Foundation ID
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-12-update1.adoc]: @divyabhushan let's not mention the API in the title
```suggestion
==== Filter Defenders by TAS Foundation ID
```

File: compute/rn/release-information/release-notes-22-12-update1.adoc (hunk 1 above)
Comment (divyabhushan-okta) [reply compute/rn/release-information/release-notes-22-12-update1.adoc]: @grevach Thanks. Updated the tentative date for now.

File: compute/rn/release-information/release-notes-22-12-update1.adoc (hunk 4 above)
Comment (Pubs-MV) [reply compute/rn/release-information/release-notes-22-12-update1.adoc:45]: @rcaballeromx is adding it to the known issues, I believe.

**Pull request merged**